src/main/java/com/beerjournal/breweriana/item/rating/ItemIdAndRatings.java

```diff
@@ -0,0 +1,15 @@
+package com.beerjournal.breweriana.item.rating;
+
+import com.beerjournal.breweriana.item.rating.persistence.Rating;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.bson.types.ObjectId;
+
+import java.util.Set;
+
+@RequiredArgsConstructor
+@Data
```

Comment:
Does it have any particular advantage besides one less annotation? I think this is quite simple case when we don't need factory method very much.
I am even willing to change it but I'd like to know the benefits well. ;)